questionnaire/2019/questions.yaml

```diff
@@ -217,6 +217,55 @@
     - Asia
     - Australia, Oceania or Pacific Islands
 
+# ----------------------------------------------------
+- section: TRANSITION TO NETWORK AUTOMATION
+  questions:
+  - title: For how long have you been leveraging automation in your network
+    type: Single choice 
+    responses:
+    - Less than 1 year
+    - 1 to 2 years
+    - 2 to 5 years 
+    - More than 5 years
+
+  - title: What actions did your team took to transition to network automation
+    type: Multiple Choice 
+    responses:
+    - Some of us learned on their own
+    - Some of us took some network automation / programming training
+    - We engaged a consulting firm to help get started
+    - We hired a dedicated Network Automation Engineer
+    - We participated in hackathon
+
+  - title: > 
+      If you (personnaly) have transitioned from a network engineer role to a network automation role, 
+      how long did it took you to make the transition 
+    type: Single Choice 
+    responses:
+    - Less than 1 year
+    - 1 to 2 years
+    - 2 to 5 years 
+    - More than 5 years
+  
+  - title: > 
+      If you (personnaly) have transitioned from a network engineer role to a network automation role, 
+      How many hours did you put in training / self learning
+    type: Single Choice 
+    responses:
```

Comment:
These answers might be better measured in ranges like our other questions. I also think we should provide a reference like "500 hours is roughly 3 months of full-time work".
```
  - 0 - 500
  - 500 - 2000
  - 2000 - 5000
  - More than 5000
```